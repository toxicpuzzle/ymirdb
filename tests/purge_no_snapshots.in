SET c 7
SET b 3 5 c
SET a 1 2 b c
SET d a b c
LIST ENTRIES
DEL a
PURGE c
PURGE b
PURGE d
PURGE a
PURGE b
PURGE a
PURGE b
LIST ENTRIES
BACKWARD c
bye