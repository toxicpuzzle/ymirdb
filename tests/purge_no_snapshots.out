> ok

> ok

> ok

> ok

> d [a b c]
a [1 2 b c]
b [3 5 c]
c [7]

> not permitted

> not permitted

> not permitted

> ok

> ok

> ok

> ok

> ok

> c [7]

> nil

> bye